src/styles/components/_Buttons.scss

```diff
@@ -0,0 +1,116 @@
+.Btn {
+	@include font-heading;
+	@include text-std;
+	border: 1px solid $hm-primary;
+	padding: #{ $base-line-height / 2 } 30px;
+	background: $hm-primary;
+	color: #FFF;
+	margin-bottom: $base-line-height / 2;
+	margin-right: $base-line-height / 2;
+	display: inline-block;
+	text-align: center;
+	cursor: pointer;
+	border-radius: 2px;
+	vertical-align: middle;
+
+	&:link,
+	&:visited {
+
+		color: #FFF;
+		text-decoration: none;
+
+		&:hover,
+		&:focus {
+			text-decoration: none;
```

Comment:
We can probably remove this `&:hover` block as text decoration is declared on line 34